New Solution/HW.04_Task1/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace HW._04_Task1
+{
+    class Program
+    {
+         static void Main(string[] args)
+            {
+                char Z = '\u005A';
```

Comment:
in c# we do not use the local variables with Upper case letter as a first letter